google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/StreamWriter.java

```diff
@@ -147,11 +152,11 @@ long getInflightWaitSeconds(StreamWriter streamWriter) {
       return connectionWorker().getInflightWaitSeconds();
     }
 
-    TableSchema getUpdatedSchema() {
+    TableSchemaAndTimestamp getUpdatedSchema(StreamWriter streamWriter) {
```

Comment:
Sorry, I mean this is actually a breaking change? Dataflow will use this method.

Reply:
Discussed offline, let's used timestamp on streamwriter when returning schema